caravel/assets/visualizations/mapbox.jsx

```diff
@@ -0,0 +1,336 @@
+// JS
+var d3 = window.d3 || require('d3');
+
+// CSS
+require('./mapbox.css');
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ScatterPlotOverlay from 'react-map-gl/src/overlays/scatterplot.react.js';
+import Immutable from 'immutable';
+import supercluster from 'supercluster';
+import ViewportMercator from 'viewport-mercator-project';
+
+const earthCircumferenceKm = 40075.16;
+
+class ScatterPlotGlowOverlay extends ScatterPlotOverlay {
+  _kmToPixels(kilometers, latitude, zoomLevel) {
```

Comment:
`per` seems more fitting for a multiplier constant whilst `to` I think is better for a function. As in given kilometers, the function converts it `to` pixels. Which reminds me, I should change `milesToKm` to `milesPerKm`